plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -80,9 +81,25 @@ var FbListInlineEdit = new Class({
 		
 		// Click outside list clears down selection
 		window.addEvent('click', function (e) {
+			if(typeof this.activeElement !== 'null') {
```

Comment:
this is wrong most likely it should be:
if (this.activeElement !== null)
as typof(null) === 'object'